i18n/fr.yaml

```diff
@@ -5,74 +5,62 @@ twitter_description:
 related_reading_2:
-  other: "Why is Exposure Notification important?"
+  other: "Pourquoi la notification d'exposition est-elle importante ?"
 related_reading_3:
-  other: "Why use Apple and Google’s technology?"
+  other: "Pourquoi utiliser la technologie d'Apple et de Google ?"
 privacy_title:
-  other: "Privacy is our top priority"
+  other: "La confidentialité est notre priorité absolue"
 privacy_content:
-  other: "<strong>COVID Shield takes a privacy-first approach to exposure notification.</strong> The COVID Shield app, its server, and the web-based results portal are all designed to preserve personal privacy while helping people safely return to work."
+  other: "<strong>L'approche de la notification d'exposition adoptée par COVID Shield place la confidentialité au rang de priorité.</strong> L'application COVID Shield, son serveur et le portail web de résultats sont tous trois conçus pour protéger la vie privée tout en aidant les gens à reprendre le travail en toute sécurité."
 privacy_action:
-  other: "Read our sample privacy policy"
+  other: "Lire notre politique de confidentialité"
 three_parts:
-  other: "COVID Shield is made up of three parts: a mobile app, a server, and a
-    web-based results portal."
+  other: "La solution COVID Shield est formée par trois composantes : une application mobile, un serveur et un portail web de résultats."
 mobile_app_title:
-  other: "Mobile app (iOS and Android)"
+  other: "Application mobile (iOS et Android)"
 mobile_app_content:
-  other: "The mobile app runs in the background and requires no user interaction after onboarding. It uses Bluetooth to collect and share random IDs with nearby phones with COVID Shield installed. If a user tests positive for COVID-19, they can choose to anonymously share their data so others can be notified of possible exposure. Sharing of random IDs is voluntary and only possible with a positive test result confirmed by a health care professional. COVID Shield periodically downloads the shared random IDs from the server and compares them on each user’s device to determine if a possible exposure has occurred."
+  other: "L'application mobile fonctionne en arrière-plan et n'exige aucune interaction avec l'utilisateur après l'intégration. Elle utilise le Bluetooth pour recueillir et partager des identifiants aléatoires avec les téléphones situés à proximité, sur lesquels COVID Shield est installé. Si un utilisateur est testé positif au COVID-19, il peut choisir de partager anonymement ses données de manière que d'autres puissent être avisés d'une possible exposition. Le partage des identifiants aléatoires se fait sur une base volontaire et n'est possible que si le résultat positif du test a été confirmé par un professionnel de santé. COVID Shield télécharge périodiquement les identifiants aléatoires partagés depuis le serveur et les compare sur l'appareil de chaque utilisateur pour déterminer si une exposition possible a eu lieu."
```

Comment:
Couple of things in this paragraph. 
**1-** "intégration" doesn't make much sense for an app onboarding (_vs_ an in-person onboarding), I suggest "processus d'accueil"
**2-** "testé positif" is an anglicism, we use "reçoit un résultat de test positif"
**3-** we say "à la" COVID-19, and not "au" COVID-19
So this paragraph becomes:
`L'application mobile fonctionne en arrière-plan et n'exige aucune interaction avec l'utilisateur après le processus d'accueil. Elle utilise le Bluetooth pour recueillir et partager des identifiants aléatoires avec les téléphones situés à proximité, sur lesquels COVID Shield est installé. Si un utilisateur reçoit un résultat de test positif à la COVID-19, il peut choisir de partager anonymement ses données de manière que d'autres puissent être avisés d'une possible exposition. Le partage des identifiants aléatoires se fait sur une base volontaire et n'est possible que si le résultat positif du test a été confirmé par un professionnel de santé. COVID Shield télécharge périodiquement les identifiants aléatoires partagés depuis le serveur et les compare sur l'appareil de chaque utilisateur pour déterminer si une exposition possible a eu lieu.`